asyncio_mqtt/client.py

```diff
@@ -264,9 +333,19 @@ def _pending_call(self, mid, value):
             #
             # However, if the callback doesn't get called (e.g., due to a
             # network error) we still need to remove the item from the dict.
-            self._pending_calls.pop(mid, None)
+            try:
```

Comment:
I personally prefer the shorter `pending_dict.pop(mid, None)` compared to the four lines of `try..except`. I guess that `try..except` is more explicit, though. It'll do. 👍

Reply:
Right, so that's because the values in `pending_dict` can't be `None`, so the default causes a type error. Bit silly since we're not assigning it to anything, but there we go. I had to change it to this, although I agree using pop is nicer.